app/src/sharedTest/java/org/oppia/android/app/databinding/TextViewBindingAdaptersTest.kt

```diff
@@ -0,0 +1,350 @@
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
+      setProfileLastVisitedText(
+        textView,
+        /* setText= */ getCurrentTimeMs - 3600000
+      )
+      assertThat(textView.text.toString()).isEqualTo("Last used an hour ago")
+    }
+  }
+
+  @Test
+  fun testSetProfileLastVisitedText_forTimeTwoHourAgo_setsLastUsedTwoHoursAgoText() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
+      setProfileLastVisitedText(
+        textView,
+        /* setText= */ getCurrentTimeMs - 7200000
+      )
+      assertThat(textView.text.toString()).isEqualTo("Last used 2 hours ago")
+    }
+  }
+
+  @Test
+  fun testSetProfileLastVisitedText_forTimeADayAgo_setsLastUsedYesterdayText() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
+      setProfileLastVisitedText(
+        textView,
+        /* setText= */ getCurrentTimeMs - 86400000
+      )
+      assertThat(textView.text.toString()).isEqualTo("Last used yesterday")
+    }
+  }
+
+  @Test
+  fun testSetProfileLastVisitedText_forTimeTwoDaysAgo_setsLastUsedTwoDaysAgoText() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
+      setProfileLastVisitedText(
+        textView,
+        /* setText= */ getCurrentTimeMs - 172800000
+      )
+      assertThat(textView.text.toString()).isEqualTo("Last used 2 days ago")
+    }
+  }
+
+  @Test
+  fun testSetProfileLastVisitedText_forTimeInFuture_setsLastUsedRecentlyText() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    fakeOppiaClock.setCurrentTimeMs(TIMESTAMP)
+    val getCurrentTimeMs = fakeOppiaClock.getCurrentTimeMs()
+    activityRule.scenario.onActivity {
+      val textView: TextView = it.findViewById(R.id.test_text_view)
+      setProfileLastVisitedText(
+        textView,
+        /* setText= */ getCurrentTimeMs + 172800000
+      )
+      assertThat(textView.text.toString()).isEqualTo("Last used recently")
+    }
+  }
+
+  @Test
+  fun testSetProfileLastVisitedText_forTimeZero_setsLastUsedRecently() {
```

Comment:
@BenHenning Is this fun name suitable?

Reply:
Yup, this is clear--thanks for checking!